### ISSUE TITLE
VSCode integrated terminal process is terminated 6) SIGABRT

### ISSUE THREAD

**Issue by josuebustos** [microsoft/vscode#104216]
- VScode: Version: 1.47.3

- OS Version: macOS Catalina: Version 10.15.6

Does this issue occur when all extensions are disabled?: No

**Purpose:**  I am trying to create a Node JS CLI tool. This tool calls exec and spawn Node JS methods.

**I can successfully:** 
A) Launch the Samsung TIzen emulator using the macOS terminal and the process continues. AND
B) I can launch the emulator using  VSCode with the flag enabled:
```
 /Applications/Visual\ Studio\ Code.app/Contents/MacOS/Electron --disable-extensions
```

**Steps to Reproduce:**
1.  I disabled all the extensions manually by hand. This did not work.
2. I tested the command in macOS, VSCode, and in VSCode Insiders terminals. Not of these worked without using the Flag.
3. Using the Node JS spawn method. I included this property ``` maxBuffer : 500 * 1024 ```. No change.
4. I also included these properties in the code ```detach: true``` and ``` stdio: 'ignore' ``` No change.
5. I attempted to detach and put the process in the background manually in Node JS code using this hack: 
```"nohup /some/cmd/here </dev/null >/dev/null 2>&1 &"``` 
 . No change. This methods works with successful method A) and B) above.

The code that's being executed in Node JS below:
```
const cmd = "/shell/cmd"; 
     const cp = spawn(cmd, [], {          
          shell: true ,
          detached: true,
          stdio: [ 'ignore', 'ignore', 'ignore' ]        
     });
cp.unref();
````

**Comment by Tyriar** [comment]
Are you trying to run VS Code within the Tizen emulator?

**Comment by josuebustos** [comment]
@Tyriar 

I found a clue to my problem from this issue on [GitHub](https://github.com/microsoft/vscode/issues/95062) and [here](https://github.com/microsoft/vscode/pull/95832). There are various open issues regarding this topic. I only included two because there are so many.

It turns ou that VS Code needs Camera and or Audio permissions in order for my script to launch a Tizen Emulator without crashing.  Now it makes sense why it only works using the Terminal.app.

Also, the shell command works just fine using VS Code integrated terminal when I launch "VS Code executable" using the Terminal.app.

Below are the specs for the VS Code version:

Version: 1.48.2
Commit: a0479759d6e9ea56afa657e454193f72aef85bd0
Date: 2020-08-25T10:09:08.021Z
Electron: 7.3.2
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Darwin x64 19.6.0
